Getting-Started/Setup/Upgrading/upgrading-to-v8.md

```diff
@@ -0,0 +1,80 @@
+
+## What will happen?
+
+When the migrations are running, Umbraco will go through your entire Umbraco 7 database and update it to the format required for Umbraco 8. The schema will be remodelled and transformed into the correct format and your existing compatible data will also be transformed to fit with how Umbraco 8 is storing data.
+
+Please be aware that these migrations will be running directly on your database. They are transforming schema and data - not transferring. Therefore always ensure that you have a backup before attempting to do this. In case something goes wrong, you will be able to rollback and try again.
+
+## How does it work?
+
+In the following guide we will try to migrate the content of an Umbraco 7.13.1 site to Umbraco 8.1.0.
+
+### Step 1: Upgrading to 7.14
+
+Before the content migration can start the site has to be 7.14+. A good idea would be to **always take a database backup** before doing an upgrade, and then check the [version specific upgrade instructions](version-specific.md).
+
+The site in this example is an Umbraco 7.13.1 site, and we will use Nuget to update it.
+
+It is a fairly simple site, with a bit of content:
+
+![v7 site with content](images/v7-content.png)
+
+Following the [general upgrade instructions](general.md) we will now upgrade via Nuget until we get to this point:
+
+![Upgrading to v7.14](images/upgrading-7.14.png)
+
+Once it is upgraded and you have verified everything is working, move on to step 2.
+
+### Step 2: Migrating content to Umbraco 8
+
+First thing to do is spin up a fresh new Umbraco 8.1+ site. Make sure it all works and that no content is there (_hint:_ choose not to install starter kit).
+
+![Fresh 8.1 site](images/fresh-8.1-site.png)
+
+Now you should take a backup of your database from the **Umbraco 7.14 site**. You can then add the information for the backup database and add that to the connection string for the  **Umbraco 8.1 site**. Do note that if you are running SQL CE, you will have to copy the database over to the new site as well.
+
+Once the connection string is set, the final step is to change the Umbraco version number in the web.config on the **Umbraco 8.1 site**, to say 7.14.0. This will mean that it thinks there is an upgrade as the Umbraco dlls are 8.1 but the config says 7.14 it needs to run the upgrade.
+
+![Set Umbraco version in the web.config](images/set-umbraco-version.png)
+
+When you start the site it will ask you to login and then show you this screen:
+
+![Upgrade database to 8.1](images/upgrade-to-8.1.png)
+
+That is all! Now the automatic migration will take over, and after a little bit you can log in and see your content:
+
+![Content is on 8.1](images/content-on-8.1.png)
+
+:::note
+Please be aware this is just a content migration. If you go to the frontend after doing this nothing will work. You will need to create all templates and other implementation again!
```

Comment:
is it clear what is left to do now...?
worth adding - migrate data from old V7 packages to V8 equivalents. Recreate all templates and any custom implementation.

I'd move the exclaimation mark to 
nothing will work! 
and leave off the 
again! 

Maybe link to the docs on the new template syntax here too